### PR TITLE
travis: use native PHP 7 nightly

### PR DESCRIPTION
**Pull request by TomasVotruba**
- Travis has it's own PHP 7 now

Among others:

- self-update does not have any real added value, managed by Travis per minor PHP version releases
- `--dev` option is by default for couple of months

### CODE REVIEW

**Comment by TazeTSchnitzel** [comment]
Thanks for doing this! I was planning on doing this after seeing the PHP 7 support announcement, but must've forgotten.

File: .travis.yml [hunk 1]
```diff
@@ -1,24 +1,11 @@
...
-    - PATH=`pwd`:$PATH composer install --dev --prefer-source --ignore-platform-reqs
+    - composer install --prefer-source --ignore-platform-reqs
 
 script:
-    # use PHP 7 for the actual tests
-    - PATH=`pwd`:$PATH make test
+    - make test
 
 php:
   - 5.6 # lol
```
Comment (bshaffer) [inline .travis.yml]: YOLO only needs to run on the latest and greatest, so this line should be removed.

BC is for the swagless.

**Comment by TazeTSchnitzel** [comment]
Why was this closed?

**Comment by TomasVotruba** [comment]
I'm cleaning up old PR that has no attention.

**Comment by TazeTSchnitzel** [comment]
It's useful and would've been merged eventually.

**Comment by TomasVotruba** [comment]
Let me know then

**Pull request closed without merging**